DataConnectors/Ubiquiti/Connector_Ubiquiti_agent.json

```diff
@@ -0,0 +1,165 @@
+{
+    "id": "Ubiquiti",
+    "title": "Ubiquiti",

```

Comment:
Please have title as Ubiquity UniFi and other relevant places in description can call this accordingly too.

Reply:
updated